package.json

```diff
@@ -58,7 +59,7 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.25.2",
     "husky": "^7.0.4",
-    "jsdoc": "^3.6.7",
+    "jsdoc": "^4.0.0",
```

Comment:
I'm a bit scared this might break things. V4 is new and they don't have documentation for it. Is there a problem between jsdoc 3 and Node 18? It seemed to be working fine for me 🤔

Reply:
Update: I am getting errors now 🤔